### DISCUSSION TITLE
Scheduling doesn't work intuitively

### DISCUSSION THREAD

**Discussion by kirtr** [borgbase/vorta#875] [General]
I've just installed Vorta and have been testing it for an automatic backup solution. To make a long story short, it took a while to figure out that Vorta was handling firing off backup jobs with an internal scheduler and not any sort of operating system scheduler. Thus, the desktop needs to be logged into and Vorta started for scheduled backups to occur.

It does not say this anywhere in the documentation. Let's get that added to the usage page.

**Comment by m3nu** [comment]
This never came up before, but do share what you would suggest to add.

**Reply by kirtr** [reply]
Hi M3nu. Here's what I'd recommend:
On the Usage page https://vorta.borgbase.com/usage/ create a section & Page for scheduling.

Add words like these:

---

Daily or periodic backups work best when the computer is programmed to  perform them. Vorta has an integrated scheduling abilities facilitate these automatic backups. After configuring your backup schedule , click the Apply button, finally inspect the Next Backup time to that Vorta has picked up your next backup time properly.

<Image of a scheduled backup and the Next Backup time here. Put mouse by Next Backup time to highlight it>

Since the Vorta application itself fires off our scheduled backups it must be running for that to occur. There is a check box on the miscellaneous tab to enable Vorta to startup automatically when you log into your desktop.

<Image of the menu page with the mouse highlighting the automatically start vorta on login option>

---

I'm thinking about guiding the folks that I support to use the Vorta + Borgbase combination and am anticipating the sorts of things that will ease adoption.

Another thought. Some applications have links to the documentation webpage for the section you are in. Sometimes they are visible links and sometimes just little blue circles with an 'i' in them that are links to a webpage. This functionality makes applications easier for people to wrap their heads around. 

Thank you for your openness to improvements, m3nu.

-Kirt

**Reply by m3nu** [reply]
Hi Kirt,

I've extended the [Usage](https://vorta.borgbase.com/usage/) page a bit to add your *Scheduling* info and provide a high-level overview of the other main functions. Basically an extended TOC.

Let me know if you would change anything. You can also edit the documentation directly on Github: https://github.com/borgbase/vorta.borgbase.com

**Comment by guiand888** [comment]
I came here looking for this after observing that my scheduled backups were not running most of the time.
Is there any way to trigger backups through systemd or a cron entry? It seems to me it might be more reliable.

**Reply by m3nu** [reply]
Can. `vorta --create PROFILE`

There is also a rewrite of the scheduler in progress that will e.g. give the option to catch up with missed backups.

https://github.com/borgbase/vorta/pull/908
